app/main.py

```diff
@@ -1,3 +1,5 @@
+from app.knights.knight import Knight
+
 KNIGHTS = {
```

Comment:
It would be meaningful to place the `KNIGHTS` config in a separate file to not overload the logic of the `main.py` file.